src/js/components/molecules/cards/card-download/stories/card-download.md

```diff
@@ -0,0 +1,4 @@
+# Card Download
+
+## DSM
+* [molecules/card-download](https://ultimaker.invisionapp.com/dsm/ultimaker/ultimaker-com/asset/components/)
```

Comment:
Can you already link the documentation? Fine for me if it is out of scope for the current PR.